src/cloud/release-notes/backward-incompatible-changes.md

```diff
@@ -48,8 +51,9 @@ The following table summarizes CLI command changes in {{ site.data.var.ct }} v20
 `vendor/bin/ece-tools patch` | `vendor/bin/ece-patches apply`
 `vendor/bin/ece-tools docker:build` | `vendor/bin/ece-docker build:compose`
 `vendor/bin/ece-tools docker:config:convert` | `vendor/bin/ece-docker  image:generate:php`
+`build_options.ini` file | Use the [`.magento.env.yaml`]({{ site.baseurl }}/cloud/project/magento-env-yaml.html)) file to configure environment variables to manage build and deploy actions across all your environments.<br><br>If you build a Cloud environment that includes the `build_options.ini` file, the build fails.
```

Comment:
This was removed somewhere else. I didn't think this file was used anymore. Do we still have to mention it?